portfolio/src/main/webapp/script.js

```diff
@@ -12,17 +12,16 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-/**
- * Adds a random greeting to the page.
- */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+// Adds a random fun fact to the page.
```

Comment:
nit: function comments generally use the /** */ notation rather than // (meant for inline comments). But in general, since your function name is self-descriptive (good job btw!), we don't actually need a comment here

https://google.github.io/styleguide/jsguide.html#jsdoc-method-and-function-comments